website/docs/api/hierarchy.mdx

```diff
@@ -325,7 +325,7 @@ function example() {
 
 Uncompacts the set `compactedSet` of indexes to the resolution `res`. `h3Set` must be at least of size `uncompactCellsSize(compactedSet, numHexes, res)`.
 
-Returns 0 on success.
+Returns 0 (`E_SUCCESS`) on success.
```

Comment:
`... or non-zero error code on failure`? Ideally we'd list out known errors here, linking to the error docs.

Reply:
Still thinking about how to add the known failure codes in here, but I agree it would be a useful addition.